ga4gh/datamodel/variants.py

```diff
@@ -474,17 +474,24 @@ def buildMetadata(
         ret = []
         header = varFile.header
         ret.append(buildMetadata(key="version", value=header.version))
-        for formatKey, value in header.formats.items():
-            if formatKey != "GT":
-                ret.append(buildMetadata(
-                    key="FORMAT.{}".format(value.name), type=value.type,
-                    number="{}".format(value.number)))
-                # NOTE: description is not currently implemented as a member
-                # of VariantMetadata in pysam/cbcf.pyx
-        for infoKey, value in header.info.items():
-            ret.append(buildMetadata(
-                key="INFO.{}".format(value.name), type=value.type,
-                number="{}".format(value.number)))
-            # NOTE: description is not currently implemented as a member
-            # of VariantMetadata in pysam/cbcf.pyx
+        formats = header.formats.items()
+        infos = header.info.items()
+        # TODO: currently ALT field is not implemented through pysam
+        # NOTE: contigs field is different between vcf files,
+        # so it's not included in metadata
+        # NOTE: filters in not included in metadata unless needed
+        keys = ["FORMAT", "INFO"]
+        for i, content in enumerate([formats, infos]):
```

Comment:
Since we're only using `i` to access the `keys` variable, it would be better to have something like
```python
for prefix, content in [("FORMAT", formats), ("INFO", infos)]:
     # same as before except we replace `keys[i]` with `prefix`.
```